examples/flutter_gallery/lib/demo/material/text_form_field_demo.dart

```diff
@@ -190,49 +190,60 @@ class TextFormFieldDemoState extends State<TextFormFieldDemo> {
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: <Widget>[
                   const SizedBox(height: 24.0),
-                  TextFormField(
-                    textCapitalization: TextCapitalization.words,
-                    decoration: const InputDecoration(
-                      border: UnderlineInputBorder(),
-                      filled: true,
-                      icon: Icon(Icons.person),
-                      hintText: 'What do people call you?',
-                      labelText: 'Name *',
+                  // Since the fields are in a Column, it's easier to
+                  // uses the non-lazy version.
```

Comment:
In this case since we're using a `Column`, we can just wrap the `Column` inside an `Autofill` and omit `clientListBuilder`